Discovery/Discovery/Presentation/DiscoveryView.swift

```diff
@@ -8,6 +8,8 @@
 import SwiftUI
 import Core
 import Theme
+import Authorization
```

Comment:
Feature modules do not need to know about each other.

Reply:
@volodymyr-chekyrta  Then what is the way to handle it because if I don't import it, it throughs an error like, for this case, `Cannot find 'LogistrationBottomView' in scope` whereaas the frameworks are already linked with eachother

<img width="874" alt="image" src="https://github.com/openedx/openedx-app-ios/assets/5606473/8f0d6795-a73e-438d-a015-47b975850055">

Reply:
We must decouple the modules from each other and leave them independent to reuse them more flexibly in the future.